File: play-2.4/swagger-play2/app/play/modules/swagger/RouteWrapper.java
package play.modules.swagger;

import play.routes.compiler.Route;

import java.util.Map;


public class RouteWrapper {

    private Map<String, Route> router;

    public RouteWrapper(Map<String, Route> router) {
        this.router = router;
    }

    public Route get(String routeName) {
        return router.get(routeName);
    }

    public boolean exists(String routeName) {
        return router.containsKey(routeName);
    }

    public Map<String, Route> getAll() {
        return router;
    }


}

File: play-2.7/swagger-play2/src/main/scala/play/modules/swagger/RouteWrapper.scala
package play.modules.swagger

import play.routes.compiler.Route

class RouteWrapper(var router: Map[String, Route]) {
  def get(routeName: String): Option[Route] = router.get(routeName)

  def apply(routeName: String): Route = router(routeName)

  def exists(routeName: String): Boolean = router.contains(routeName)

  def getAll: Map[String, Route] = router
}
